wiki/Modding/en.md

```diff
@@ -1,98 +1,111 @@
+**Modding** is the process of examining another's [beatmap](/wiki/Beatmaps) and making suggestions. These suggestions can be either to fix errors or to simply improve the map. The person modding the map is called a "modder", and the person who made the map is referred to as the "mapper". Modding can be useful in three main ways:
+
+1.  If your mod post is useful to the mapper, then they will give you [kudosu](/wiki/kudosu). Kudosu can be used to boost your map's star priority up.
+2.  "Mod4mods", or when you make a deal to mod somebody else's map if they mod yours, is a great way to get your map modded.
+3.  A good mod post reduces the amount of time the approval teams spend pointing out basic mistakes. This in turn makes the approval teams less stressed, and that makes them more likely to mod your map. Happy [BNs](/wiki/People/Beatmap_Nomination_Group) are typically in a better mood than a BN who just spent 10 minutes crying because of a map that clearly needed a lot more modding.
 
 If you are learning how to map or already know how to map, modding can be a great way to improve mapping skills.
 
-Basic modding method
---------------------
+## Basics
 
-First, download a map from the [beatmap](beatmap "wikilink") thread, which can be located in either the [Pending](Pending_Beatmap "wikilink") or [Work In Progress/Help](Works_In_Progress/Help "wikilink") forum (not the [Ranked](Ranked_beatmap "wikilink")/[Approved](Approved_Beatmap "wikilink") Beatmaps forum!). You can either randomly choose maps from these sub-forums to mod or you can open a modding queue in the **Modding Queue** sub-forum under the [Work In Progress/Help](Works_In_Progress/Help "wikilink") forum and accept requests. If you are opening a queue, it is a good idea to see how these queues work by reading some threads in the 'Modding Queues' sub-forum.
+First, download a map from the [beatmap](https://osu.ppy.sh/forum/12) thread, which can be located in either the [Pending](https://osu.ppy.sh/forum/6) or [Work In Progress/Help](https://osu.ppy.sh/forum/10) forum (not the [Ranked/Approved](https://osu.ppy.sh/forum/14) beatmaps forum!). You can either randomly choose maps from these subforums to mod or you can open a modding queue in the 'Modding Queue' subforum under the Work In Progress/Help forum and accept requests. If you are opening a queue, it is a good idea to see how these queues work by reading some threads in the 'Modding Queues' subforum.
 
-Next, go into the [Editor](Editor "wikilink") and select the map which you want to mod. Then, start up a reply to the [Beatmap's](Beatmap "wikilink") thread.
+Next, go into the [editor](/wiki/Beatmap_Editor) and select the map which you want to mod. Then, start up a reply to the [beatmap's](/wiki/Beatmaps) thread.
 
 Choose any of the difficulties to begin your mod. It is recommended to start with the easiest or hardest difficulty and make your way up or down the different difficulties.
```

Comment:
`Choose any of the difficulties to start a mod. It is recommended to start with the easiest or hardest difficulty and continue up or down the different difficulties.`